unit_tests/nemesis/test_registry.py

```diff
@@ -7,13 +7,17 @@
 from sdcm.nemesis_registry import NemesisRegistry
 
 
-class TestNemesis:
-    COMMON_STRING = "called test function "
+class FlagClass:
     flag_a = False
     flag_b = False
     flag_c = False
     flag_d = False
     flag_common = False
+    flag_true = True
+
+
+class TestNemesis(FlagClass):
```

Comment:
Intuitively I would expect the `CustomNemesisD` to be renamed to the `CustomNemesisAD`